README.md

```diff
@@ -2121,8 +2121,8 @@ force-exclude = true
 The style in which violation messages should be formatted: `"text"`
 (default), `"grouped"` (group messages by file), `"json"`
 (machine-readable), `"junit"` (machine-readable XML), `"github"`
-(GitHub Actions annotations) or `"gitlab"`
-(GitLab CI code quality report).
+(GitHub Actions annotations), `"gitlab"`
+(GitLab CI code quality report) or `\"pylint\"` (Pylint text format).
```

Comment:
```suggestion
(GitLab CI code quality report) or `"pylint"` (Pylint text format).
```

I don't think the backslashes are needed.

<img width="395" alt="image" src="https://user-images.githubusercontent.com/1556054/213443817-02dfbe47-08e0-448d-90fc-72d3c61e4f9f.png">